packages/dds/tree/src/shared-tree/schematizingTreeView.ts

```diff
@@ -511,7 +511,7 @@ export function requireSchema(
 	schemaPolicy: FullSchemaPolicy,
 ): CheckoutFlexTreeView {
 	const slots = checkout.forest.anchors.slots;
-	assert(!slots.has(ContextSlot), 0x8c2 /* Cannot create second view from checkout */);
+	assert(!slots.has(ViewSlot), 0x8c2 /* Cannot create second view from checkout */);
```

Comment:
TreeView use this code path to make their contexts, so this assert will fail when trying constructing a CheckoutFlexTreeView and Context for a TreeView if you make this change because the TreeView exists.

Reply:
I believe the check was correct how it was.

Reply:
Note view in the assert message is CheckoutFlexTreeView (see the return type of this method), and view in ViewSlot is TreeView (see its type definition). Perhaps the message should be more clear